frontend/src/service/ShortHTTP.api.ts

```diff
@@ -16,6 +18,13 @@ export class ShortHTTPApi {
     return this.httpService.getJSON<boolean>(url);
   }
 
+  getAuthFeatureToggle(featureID: string): Promise<boolean> {
+    const url = `${this.baseURL}/features/${featureID}`;
+    const headers = { Authorization: this.authService.getAuthBearerToken() };
+
+    return this.httpService.getJSON<boolean>(url, headers);
+  }
```

Comment:
I think we should replace `getFeatureToggle` with `getAuthFeatureToggle` because they are actually doing the same thing. Auth is the internal implementation of toggle itself.